engine/src/render/deferred_shading/plugin.cpp

```diff
@@ -337,13 +344,81 @@ void cubos::engine::deferredShadingPlugin(Cubos& cubos)
+                                        lightLocalToWorld.mat, glm::vec3(1.0F / lightLocalToWorld.worldScale())));
+                                    break;
+                                case 1:
+                                    lightView =
+                                        glm::inverse(glm::scale(glm::rotate(lightLocalToWorld.mat, glm::radians(180.0F),
+                                                                            glm::vec3(0.0F, 1.0F, 0.0F)),
+                                                                glm::vec3(1.0F / lightLocalToWorld.worldScale())));
+                                    break;
+                                case 2:
+                                    lightView =
+                                        glm::inverse(glm::scale(glm::rotate(lightLocalToWorld.mat, glm::radians(90.0F),
+                                                                            glm::vec3(0.0F, 1.0F, 0.0F)),
+                                                                glm::vec3(1.0F / lightLocalToWorld.worldScale())));
+                                    break;
+                                case 3:
+                                    lightView =
+                                        glm::inverse(glm::scale(glm::rotate(lightLocalToWorld.mat, glm::radians(-90.0F),
+                                                                            glm::vec3(0.0F, 1.0F, 0.0F)),
+                                                                glm::vec3(1.0F / lightLocalToWorld.worldScale())));
+                                    break;
+                                case 4:
+                                    lightView =
+                                        glm::inverse(glm::scale(glm::rotate(lightLocalToWorld.mat, glm::radians(90.0F),
+                                                                            glm::vec3(1.0F, 0.0F, 0.0F)),
+                                                                glm::vec3(1.0F / lightLocalToWorld.worldScale())));
+                                    break;
+                                case 5:
+                                    lightView =
+                                        glm::inverse(glm::scale(glm::rotate(lightLocalToWorld.mat, glm::radians(-90.0F),
+                                                                            glm::vec3(1.0F, 0.0F, 0.0F)),
+                                                                glm::vec3(1.0F / lightLocalToWorld.worldScale())));
+                                    break;
+                                default:
+                                    lightView = glm::inverse(glm::scale(
+                                        lightLocalToWorld.mat, glm::vec3(1.0F / lightLocalToWorld.worldScale())));
+                                    break;
+                                }
+
+                                perLight.matrices[i] = lightProj * lightView;
+                            }
```

Comment:
I think you can simplify this section of code a bit - the for barely does anything, why not unfold it and remove the switch case entirely? One other option (probably better) would be to move the values which change for each case into an array. E.g.
```cpp
static const lightViewAngle[6] = { 0.0F, 180.0F, ...};
static const glm::vec3 lightViewAxis[6] = { ... };
... // use these within the for loop, avoid the switch